base4kids2.ldif

```diff
@@ -498,6 +498,43 @@ attributeTypes: (
       DESC 'Determines whether a base4kids object is disabled but still active'
       SUP base4kidsIsBlocked )
 #
+# Name:     base4kidsTelephoneNumberBusiness
+# Syntax:   Phone number
+# Examples: +01234567890
+attributeTypes: (
+      1.3.6.1.4.1.31534.2.5.2.42
+      NAME 'base4kidsTelephoneNumberBusiness'
+      DESC 'Business telephone number'
+      EQUALITY telephoneNumberMatch
```

Comment:
Please don't redefine the syntax and matchings rules but rather simply derive it from the existing `telephoneNumber` attribute (RFC 2256 ): `SUP telephoneNumber`